src/color.rs

```diff
@@ -62,6 +63,7 @@ pub enum Theme {
 
 pub struct Colors {
     colors: Option<HashMap<Elem, Colour>>,
+    lscolors: LsColors,
```

Comment:
I think the variable should be optional. If the `LS_COLOR` is not set we should fallback into the default theme colors.